src/ui/public/vislib/visualizations/_map.js

```diff
@@ -1,28 +1,31 @@
 import _ from 'lodash';
 import $ from 'jquery';
 import L from 'leaflet';
-import marked from 'marked';
-marked.setOptions({
-  gfm: true, // Github-flavored markdown
-  sanitize: true // Sanitize HTML tags
-});
-
 import VislibVisualizationsMarkerTypesScaledCirclesProvider from 'ui/vislib/visualizations/marker_types/scaled_circles';
 import VislibVisualizationsMarkerTypesShadedCirclesProvider from 'ui/vislib/visualizations/marker_types/shaded_circles';
 import VislibVisualizationsMarkerTypesGeohashGridProvider from 'ui/vislib/visualizations/marker_types/geohash_grid';
 import VislibVisualizationsMarkerTypesHeatmapProvider from 'ui/vislib/visualizations/marker_types/heatmap';
-export default function MapFactory(Private, tilemap, $sanitize) {
+import 'ui/vislib/lib/tilemap_settings';
+
+
+import uiRoutes from 'ui/routes';
+
+let tilemapUrl = '';
```

Comment:
i am not 100% sure what spalger is suggesting but i agree that this part looks a bit weird ...